.gitlab-ci.yml

```diff
@@ -62,4 +62,4 @@ CI-Proxy-Deploy:
     - helm delete --purge db-ciservice-"$CI_COMMIT_REF_NAME"
     - helm install stable/postgresql --name db-ciservice-"$CI_COMMIT_REF_NAME" --set postgresqlDatabase=ciproxy --set persistence.enabled=false --set service.type=ClusterIP
     - export POSTGRESQL_PASSWORD=$(kubectl get secret --namespace default db-ciservice-"$CI_COMMIT_REF_NAME"-postgresql -o jsonpath="{.data.postgresql-password}" | base64 -d)
-    - helm upgrade ciproxy-"$CI_COMMIT_REF_NAME" ./charts/exciproxy --install --force --wait --timeout 3600 --set tls="$TLS" --set db.user=postgres --set db.password=$POSTGRESQL_PASSWORD --set db.name=ciproxy --set db.host=db-ciservice-"$CI_COMMIT_REF_NAME"-postgresql --set db.pool=5 --set imagePullSecrets="$CI_COMMIT_REF_NAME"secret --set yaml=https://raw.githubusercontent.com/CrossCloudCI/cncf-configuration/"$CI_COMMIT_REF_NAME"/cross-cloud.yml --set project.segment_env="$PROJECT_SEGMENT_ENV" --set ingress.host="$CIPROXY_INGRESS" --set ingress.port=4001 --set image.repository="$CI_REGISTRY_IMAGE/ciproxy-$CI_COMMIT_REF_NAME" --set image.tag="$CI_COMMIT_SHA"
+    - helm upgrade ciproxy-"$CI_COMMIT_REF_NAME" ./charts/exciproxy --install --force --wait --timeout 3600 --set tls="$TLS" --set db.user=postgres --set db.password=$POSTGRESQL_PASSWORD --set db.name=ciproxy --set db.host=db-ciservice-"$CI_COMMIT_REF_NAME"-postgresql --set db.pool=5 --set imagePullSecrets="$CI_COMMIT_REF_NAME"secret --set yaml=https://raw.githubusercontent.com/CrossCloudCI/cncf-configuration/"$PROJECT_SEGMENT_ENV"/cross-cloud.yml --set project.segment_env="$PROJECT_SEGMENT_ENV" --set ingress.host="$CIPROXY_INGRESS" --set ingress.port=4001 --set image.repository="$CI_REGISTRY_IMAGE/ciproxy-$CI_COMMIT_REF_NAME" --set image.tag="$CI_COMMIT_SHA"
```

Comment:
I think helm should use a deployment name variable and the CI_COMMIT_REF_NAME should be left alone.